exporter/elasticsearchexporter/README.md

```diff
@@ -213,3 +213,89 @@ Other metric types (Histogram, Exponential Histogram, Summary) are ignored.
 [data stream]: https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html
 [ecs]: https://www.elastic.co/guide/en/ecs/current/index.html
 [SemConv]: https://github.com/open-telemetry/semantic-conventions
+
+
+## ECS Mapping
```

Comment:
Should this be linking to semantic conventions?

Reply:
should this conversion be in the semantic-conventions specification?

Reply:
i don't think so

Reply:
I think it should. Here is the equivalent for log mapping for Splunk HEC: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model-appendix.md#splunk-hec

Reply:
i see, there's also an ECS specific mapping also defined there: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model-appendix.md#elastic-common-schema

do you suggest adding a link here pointing to that mapping

Reply:
added a backlink to this readme